src/components/Swap/Toolbar/Caption.tsx

```diff
@@ -196,9 +204,11 @@ export function Trade({ trade, outputUSDC, gasUseEstimateUSD }: TradeProps) {
   )
 }
 
-type PriceImpactWarningProps = Omit<TradeProps, 'gasUseEstimateUSD'> & { impact: PriceImpact }
+interface PriceImpactProps {
+  impact: PriceImpactValue
+}
 
-export function PriceImpactWarning({ impact }: PriceImpactWarningProps) {
+export function PriceImpact({ impact }: TradeProps & PriceImpactProps) {
```

Comment:
You don't need TradeProps here at all. You don't need to back out the changes to WrapCurrencyProps or TradeProps, though.

nit: WrapCurrency could just be called Wrap (similarly, WrapCurrencyProps->WrapProps).